Hestia/app/src/main/java/hestia/UI/activities/home/DeviceListFragment.java

```diff
@@ -44,11 +44,22 @@ public DeviceListFragment() {
         super();
     }
 
-    public DeviceListFragment(Context context, ServerCollectionsInteractor serverCollectionsInteractor) {
-        super();
-        this.context = context;
+    public static DeviceListFragment newInstance() {
+        DeviceListFragment fragment = new DeviceListFragment();
+        return fragment;
+    }
+
+    //TODO: Check whether this works with a server
```

Comment:
Commented code should not enter development anymore. I suggest fixing this or removing it altogether.